frontend/src/components/form/inputs/chips-input.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FormEvent } from "react";
+import { IonChip, IonIcon } from "@ionic/react";
+import { closeCircle } from "ionicons/icons";
+import { Item } from "@/types/item.ts";
+import './inputs.css';
+
+type ChipsProperties = {
+  label?: string;
+  required?: boolean;
+  disabled?: boolean;
+  items: Array<Item>;
+  activeItemsValues: Array<number | string>;
+  setActiveItemsValues: (value: Array<number | string>) => void;
+  onInvalid?: (e: FormEvent<HTMLInputElement>) => void
+}
+
+export const ChipsInput: React.FC<ChipsProperties> = ({
+                                                        label,
+                                                        required,
+                                                        disabled,
+                                                        items,
+                                                        activeItemsValues,
+                                                        setActiveItemsValues,
+                                                        onInvalid
+                                                      }) => {
+
+  const deactivateChip = (chip: Item) => setActiveItemsValues(activeItemsValues.filter(v => v !== chip.value))
+  const activateChip = (chip: Item) => setActiveItemsValues([...activeItemsValues, chip.value])
+
+  return <div id="aplose-input" className="chips" aria-disabled={ disabled }>
+    { label && <p id="label"
+                  className={ required ? 'required' : '' }>
+      { label }{ required ? '*' : '' }
+    </p> }
```

Comment:
General comments as it applies to several files:

- You used `id` attribute which is supposed to appear only one time on a HTML document. As inputs can be used many times, I think the page won't be syntactically correct.
- Why don't use `label` tag instead of `p`? It's semantically better.